README.rst

```diff
@@ -161,6 +161,38 @@ Client where you download an index template that is stored in the direcotry
             enabled: true
             template: elasticsearch/files/my_index_template.json
 
+Client where you download an index template from the metadata definition and force index creation:
+
+.. code-block:: yaml
+
+    elasticsearch:
+      client:
+        enabled: true
+        server:
+          host: elasticsearch.host
+          port: 9200
+        index:
+          my_index:
+            enabled: true
+            enabled: true
+            force_create: true
```

Comment:
since force_create also applies to delete, it would make sense to rename it "force" or "force_operation".